### PR TITLE
Fix documentation confusion about HEAD-SOUR-DATA

### LINKED ISSUE #549: Documentation confusion about HEAD-SOUR-DATA
In the 7.0 doc version, 

> HEAD-SOUR-DATA 
> The electronic data source or digital repository from which this dataset was exported. The payload is the name of that source, with substructures providing additional details about the source (not the export).

However, the link that takes you there says this: 
> SOUR describes the originating software.
> CORP describes the corporation creating the software.
> HEAD.SOUR.DATA describes a larger database this data is extracted from.

HEAD.SOUR.DATA does not describe a larger database this data is extracted from. It's just a name of the data source or digital repository as per the linked-to-definition. The substructures don't really provide accurate or concise information about the software in question. I keep thinking the source data is the overall "where-the-data-itself" comes from, as in, the whole genealogy. 

In my case, HEAD.SOUR.DATA (to me), is the name of a book on rootsweb. Then I want to include the link to that book on rootsweb, the substructures don't seem to support that kind of thing. Which is why I'm confused. If I use "CORP", which is inconsistently defined (as I'm not a corporation), it gets even more confusing, because I'm copying someone's book, which is in HTML format, and converting it to GEDCOM - does that make me the CORP? Or is that the original author of the book?

CORP is defined: 
> CORP (Corporate name) g7:CORP
> The name of the business, corporation, or person that produced or commissioned the product.

Questions: 
What is "this data"? 

What is "the product"? And does this apply to the CORP creating the software? Is the product the software? Is it the book I'm transcribing?

HEAD.SOUR.DATA is not consistently defined. In both places, the appropriate definition would be to say: "The name of the source software" (though I think that DATA is a bad name for a name. Better would be to rename this particular tag or create a new one as HEAD.SOUR.SOFTWARE, with a substructure for +1 CORP (name of the entity that created it) (along with its substructure - changing CORP to ENTI (entity) might be a good thing, too, as it clears away some ambiguity about whether the creator is a corporation, business, or an individual/person), +1 NAME (name of the software itself), +1 DLOC (or WWW) download location), +1 EWEB (entity website, different from download location) to remain consistent with "ENTI")

So I'm rather confused. 

What I want to be able to do: 
- Say I'm the author of the gedcom itself, via a program that I myself wrote. 
- Say where I got the original data (its URL)
- Express the name of the book
- Give credit to the original author of the book

I realize this is about 3 things in one. 
1. Please update the docs to be more consistent. 
1. I've included some suggestions for new tags and ways to handle software and related websites.
1. Request help on the proper way to handle source and credit in the header of a gedcom. 

Thanks!

### PR DESCRIPTION
**Pull request by dthaler**
* Make HEAD.SOUR.DATA wording consistent between two places in spec
* Deprecate HEAD.SOUR.DATA in favor of just using HEAD.SOUR.NAME

Fixes #549

### CODE REVIEW

**Review by tychonievich — APPROVED** [review]

File: specification/gedcom-3-structures-1-organization.md [hunk 1]
```diff
@@ -166,9 +166,12 @@ A few substructures of note:
 - `SCHMA` gives the meaning of extension tags; see [Extensions](#extensions) for more details.
 - `SOUR` describes the originating software.
     - `CORP` describes the corporation creating the software.
-    - `HEAD`.`SOUR`.`DATA` describes a larger database this data is extracted from.
+    - `HEAD`.`SOUR`.`DATA` describes a larger database, data source, or digital repository this data is extracted from.
```
Comment (dthaler) [inline specification/gedcom-3-structures-1-organization.md:169]:
```suggestion
    - `HEAD`.`SOUR`.`DATA` describes a larger database, electronic data source, or digital repository this data is extracted from.
```

File: specification/gedcom-3-structures-3-meaning.md [hunk 2]
```diff
@@ -428,9 +428,9 @@ See `g7:DATA`.
 
 #### `DATA` (Data) `g7:HEAD-SOUR-DATA`
 
-The electronic data source or digital repository from which this dataset was exported.
-The payload is the name of that source,
-with substructures providing additional details about the source (not the export).
+The electronic database, data source, or digital repository from which this dataset was exported.
```
Comment (dthaler) [inline specification/gedcom-3-structures-3-meaning.md:431]:
```suggestion
The database, electronic data source, or digital repository from which this dataset was exported.
```

File: specification/gedcom-3-structures-3-meaning.md [hunk 3]
```diff
@@ -428,9 +428,9 @@ See `g7:DATA`.
 
 #### `DATA` (Data) `g7:HEAD-SOUR-DATA`
 
-The electronic data source or digital repository from which this dataset was exported.
-The payload is the name of that source,
-with substructures providing additional details about the source (not the export).
+The database, electronic data source, or digital repository from which this dataset was exported.
+The payload is the name of the database, data source, or digital repository,
```
Comment (dthaler) [inline specification/gedcom-3-structures-3-meaning.md:432]:
```suggestion
The payload is the name of the database, electronic data source, or digital repository,
```

**Comment by dthaler** [comment]
Approved during GEDCOM Steering Committee meeting

**Pull request merged**